### ISSUE TITLE
Improvement when forking the repo

### ISSUE THREAD

**Issue by Turbots** [gm2552/hungryman#3]
Hello,

I've just forked your repository and might want to make some improvements/updates here and there, but the way the `workload.yaml` files have been setup, it means we would have to make the `git` references more generic, otherwise it's impossible to make proper pull requests. How could we do this? kustomize? YTT? regular templating or scripting?

**Comment by gm2552** [comment]
```
it means we would have to make the git references more generic
```

Are you speaking of the individual workload.yaml files in each source code directory, or the templated `workloads.yaml`.  If the previous, then this issue exists in pretty much every accelerator generated project.  I'm hearing there may be an enhancement at a later time to provide a GitRepo in the accelerator that would automatically create a GitRepo.  If the becomes a reality, the plan is update the repo reference in the workload.yaml files to this new repo.

**Comment by Turbots** [comment]
Agreed, I heard that this is on the roadmap for TAP 1.3 iirc, so we'll wait for that.
